mozilla/bedrock
# [bug 1422899] Remove old Firefox hub template

## Description
- Removes olf `/firefox/` hub template and assets.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1422899

## Testing
- Probably a good idea to run `gulp static:clean` prior to testing this PR.
- This PR moves/renames a bunch of asset paths from `/firefox/hub/quantum/` to `/firefox/home/`, so check I didn't miss any?
- Check I didn't remove any other assets that may still be needed?
- Check translations still appear as expected.
